### PR TITLE
Implement ComputerSystem for a composed resource

### LINKED ISSUE #270: Implement ComputerSystem for a composed resource
A new System should represent the composed resource (type "Composed" with info about processor, memory, disk drives and networks) - representation of a server profile instance.
Leverage current implementation. Needs to include info about disk drives and ethernet interface. Maybe we will need to review some previous mappings.

Since the toolkit does not support POST yet, a server profile can be created direclty in OneView and use UUID to build Redfish URI for testing.
Out of scope: update System collections to include Composed instances.

### PR DESCRIPTION
**Pull request by ricardogpsf**
- Removes handling of server hardware and put handling to serve profile
- Changes unit tests to the new behavior
- Adds new mockup files to the new unit tests

Closes #270

### CODE REVIEW

File: oneview_redfish_toolkit/api/computer_system.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class ComputerSystem(RedfishJsonValidator):
     SCHEMA_NAME = 'ComputerSystem'
     BASE_URI = '/redfish/v1/Systems'
 
-    def __init__(self, server_hardware, server_hardware_types):
+    def __init__(self, server_hardware, server_hardware_types, server_profile):
```
Comment (karolcatunda) [inline oneview_redfish_toolkit/api/computer_system.py]: @ricardogpsf Please, update copyright year on top.

File: oneview_redfish_toolkit/blueprints/computer_system.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def get_computer_system(uuid):
     """Get the Redfish Computer System for a given UUID.
 
         Return ComputerSystem redfish JSON for a given
-        server hardware or server profile templates UUID.
+        server profile or server profile template UUID.
```
Comment (karolcatunda) [inline oneview_redfish_toolkit/blueprints/computer_system.py]: Same comment above :+1:

File: oneview_redfish_toolkit/tests/api/test_computer_system.py [hunk 3]
```diff
@@ -40,56 +39,36 @@ def setUp(self):
         ) as f:
             self.server_hardware_types = json.load(f)
 
+        # Loading ServerProfile mockup value
```
Comment (karolcatunda) [inline oneview_redfish_toolkit/tests/api/test_computer_system.py]: Same here too :+1:

**Comment by ricardogpsf** [comment]
> I think is missing include the fixed issue on Changelog, right @ricardogpsf ?

@karolcatunda It is because we talked about this before and decided change the CHANGELOG after PRs be merge, at end of "sprint", to avoid merge conflicts and force the rebase to all others PR.

**Pull request merged**